src/pg_to_evalscript/javascript_common/common.js

```diff
@@ -0,0 +1,42 @@
+function parse_rfc3339(dt, default_h = 0, default_m = 0, default_s = 0) {
```

Comment:
It would be great if unit tests would be added for `parse_rfc3339` as well, given it's so often used. I trust it works correctly now, but just so we don't break anything in the future if we make any changes (e.g. for supporting time strings).

Reply:
some tests were added.